clippy.toml

```diff
@@ -88,9 +88,12 @@ disallowed-macros = [
     { path = "proptest::prop_oneof", reason = "use `proptest::strategy::Union::new` instead" },
     { path = "log::log", reason = "use the macros provided by `tracing` instead (database-issues#3001)" },
     { path = "tracing::instrument", reason = "use `mz_ore::instrument` instead" },
+    { path = "anyhow::anyhow", reason = "use structured errors instead" },
+    { path = "anyhow::bail", reason = "use structured errors instead" },
```

Comment:
imo this would be more valuable with a fleshed out example to point people towards